runtime/envelope/envelope.go

```diff
@@ -263,7 +263,7 @@ func (e *Envelope) RunProfiling(_ context.Context, req *protos.RunProfiling) (*p
 // runWeavelet with the provided environment and wait for it to terminate.
 //
 // It returns true,_ if some incoming argument is bad any retries should be stopped.
```

Comment:
Delete this line now that it's obsolete?

Reply:
Ah, yes! I've pushed another commit that removes this line